src/components/Button/Button.tsx

```diff
@@ -36,7 +36,7 @@ export type ButtonProps = ButtonHTMLElementProps & {
    * - **reset** The clickable is a reset clickable (resets the form-data to its initial values)
    */
   type?: 'button' | 'reset' | 'submit';
-};
+} & VariantStatus;
```

Comment:
Pull in the `VariantStatus` which pairs the different variants to the available statuses